assets/stylesheets/_components.scss

```diff
@@ -370,3 +370,12 @@
 @import 'signup/validation-fieldset/style';
 @import 'support/support-user/style';
 @import 'vip/vip-logs/style';
+
+// ==========================================================================
```

Comment:
Hmm that is a great question.  Seems fine to me to place the import here, but have to agree grabbing from `node_modules` looks/feels strange.  The [file](https://github.com/bvaughn/react-virtualized/blob/master/styles.css) isn't huge, but we could take the approach done with `component/tip` and grab the base styles and place within calypso itself [#](https://github.com/Automattic/wp-calypso/blob/dabf92fa84865576ecd8d806bd21c9cc4184e73d/client/components/popover/style.scss)